CMakeLists.txt

```
@@ -16,27 +16,13 @@ set(CMAKE_BUILD_TYPE Release)
 include(GNUInstallDirs)
 list(APPEND CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
 
+find_package(CUDAToolkit)
 find_package(cudawrappers 0.5.0 REQUIRED cu nvrtc)
```

We need to update this as soon as it is available:
```suggestion
find_package(cudawrappers 0.6.0 REQUIRED cu nvrtc)
```